apps/finance/app/src/script.js

```diff
@@ -1,4 +1,4 @@
-import Aragon from '@aragon/api'
+import Aragon, { SYNC_STATUS_SYNCING, SYNC_STATUS_SYNCED } from '@aragon/api'
```

Comment:
Perhaps an improvement in the API itself; what do you think about namespacing these constants so that we would instead do

```js
import { syncStatus } from '@aragon/api'

case syncStatus.SYNCING
```

Or similar?

Reply:
Sounds good!